src/spot.gresource.xml

```diff
@@ -50,25 +51,5 @@
   <gresource prefix="/dev/alextren/Spot/icons/scalable/actions/">
     <file compressed="true" preprocess="xml-stripblanks" alias="music-queue-symbolic.svg">app/components/selection/icons/music-queue-symbolic.svg</file>
     <file compressed="true" preprocess="xml-stripblanks" alias="playlist2-symbolic.svg">app/components/selection/icons/playlist2-symbolic.svg</file>
-  </gresource>
-	<gresource prefix="/dev/alextren/Spot/icons/scalable/actions/">
```

Comment:
oh :( are we not doing this for now? maybe later?

Reply:
Yeah, this will be shipped as a separate MR.